1.4.0 fail to build with autotools

synfig 1.4.0 Debian unstable amd64

```
libtool: compile:  g++ -DHAVE_CONFIG_H -I../.. -I../../src -Wdate-time -D_FORTIFY_SOURCE=2 -I/usr/include -I/usr/include/glibmm-2.4 -I/usr/lib/x86_64-linux-gnu/glibmm-2.4/include -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/sigc++-2.0 -I/usr/lib/x86_64-linux-gnu/sigc++-2.0/include -pthread -I/usr/include/giomm-2.4 -I/usr/lib/x86_64-linux-gnu/giomm-2.4/include -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/glibmm-2.4 -I/usr/lib/x86_64-linux-gnu/glibmm-2.4/include -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/sigc++-2.0 -I/usr/lib/x86_64-linux-gnu/sigc++-2.0/include -I/usr/include/libxml++-2.6 -I/usr/lib/x86_64-linux-gnu/libxml++-2.6/include -I/usr/include/libxml2 -I/usr/include/glibmm-2.4 -I/usr/lib/x86_64-linux-gnu/glibmm-2.4/include -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/sigc++-2.0 -I/usr/lib/x86_64-linux-gnu/sigc++-2.0/include -fopenmp -DMAGICKCORE_HDRI_ENABLE=1 -DMAGICKCORE_QUANTUM_DEPTH=16 -fopenmp -DMAGICKCORE_HDRI_ENABLE=1 -DMAGICKCORE_QUANTUM_DEPTH=16 -fopenmp -DMAGICKCORE_HDRI_ENABLE=1 -DMAGICKCORE_QUANTUM_DEPTH=16 -I/usr/include/x86_64-linux-gnu//ImageMagick-7 -I/usr/include/ImageMagick-7 -I/usr/include/x86_64-linux-gnu//ImageMagick-7 -I/usr/include/ImageMagick-7 -I/usr/include/x86_64-linux-gnu//ImageMagick-7 -I/usr/include/ImageMagick-7 -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -DGL_GLEXT_PROTOTYPES -pthread -I/usr/include/pango-1.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -I/usr/include/harfbuzz -I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16 -D_REENTRANT -I/usr/include/mlt++ -I/usr/include/mlt -I/usr/include/ETL -I/usr/include/sigc++-2.0 -I/usr/lib/x86_64-linux-gnu/sigc++-2.0/include -DWITH_OPENGL -DSYNFIG_NO_DEPRECATED -DLOCALEDIR=\"/usr/share/locale\" -DLIBDIR=\"/usr/lib/x86_64-linux-gnu\" -DSYSCONFDIR=\"/etc/synfig\" -fdebug-prefix-map=/src/synfig-1.4.0=. -fstack-protector-strong -Wformat -Werror=format-security -O2 -DNDEBUG -W -Wall -c rendering/renderer.cpp  -fPIC -DPIC -o rendering/.libs/libsynfig_la-renderer.o
In file included from rendering/opengl/task/taskgl.h:32,
                 from rendering/renderer.cpp:54:
rendering/opengl/task/../surfacegl.h:60:15: error: conflicting return type specified for 'virtual void synfig::rendering::SurfaceGL::reset_vfunc()'
   60 |  virtual void reset_vfunc();
      |               ^~~~~~~~~~~
In file included from rendering/task.h:39,
                 from rendering/optimizer.h:28,
                 from rendering/renderer.h:35,
                 from rendering/renderer.cpp:44:
rendering/surface.h:122:15: note: overridden function is 'virtual bool synfig::rendering::Surface::reset_vfunc()'
  122 |  virtual bool reset_vfunc()
      |               ^~~~~~~~~~~
In file included from rendering/renderer.cpp:54:
rendering/opengl/task/taskgl.h: In member function 'virtual synfig::rendering::Surface::Token::Handle synfig::rendering::TaskGL::get_mode_target_token() const':
rendering/opengl/task/taskgl.h:55:27: error: could not convert 'synfig::rendering::SurfaceGL::token' from 'synfig::rendering::Surface::Token' to 'synfig::rendering::Surface::Token::Handle' {aka 'synfig::ConstRef<synfig::rendering::Surface::Token>'}
   55 |   { return TargetSurface::token; }
      |                           ^~~~~
      |                           |
      |                           synfig::rendering::Surface::Token
make[3]: *** [Makefile:3510: rendering/libsynfig_la-renderer.lo] Error 1
```

--- a/synfig-core/configure.ac
+++ b/synfig-core/configure.ac
@@ -339,52 +339,6 @@ PKG_CHECK_MODULES(LIBCAIRO, [cairo >= 1.12.0],[
 	AC_MSG_ERROR([ ** You need to install cairo version 1.12.0 or higher.])
 ])
 
-AC_MSG_CHECKING([for OpenGL])
-AC_ARG_WITH(opengl,
-	AS_HELP_STRING(--with-opengl, [enable OpenGL (default=no)]),[
-],[
-	with_opengl="no"
-])
-if test $with_opengl != "no" ; then {
-	case "$host" in
-		*darwin*)
-			AC_MSG_RESULT([ ** OpenGL is not available for OSX yet. Disabled.])
-			with_opengl="no"
-		;;
-		*)
-			PKG_CHECK_MODULES(LIBGL, gl, [
-				CONFIG_DEPS="$CONFIG_DEPS gl"
-				LIBGL_CFLAGS="$LIBGL_CFLAGS -DGL_GLEXT_PROTOTYPES"
-				CONFIG_CFLAGS="$CONFIG_CFLAGS -DWITH_OPENGL"
-				with_opengl="yes"
-			],[
-				AC_MSG_RESULT([ ** Cannot find gl.pc. OpenGL disabled.])
-				with_opengl="no"
-			])
-		;;
-	esac
-} ; fi
-AM_CONDITIONAL(WITH_OPENGL, test $with_opengl = yes)
-
-AC_ARG_WITH(opencl,
-	AS_HELP_STRING(--with-opencl, [enable OpenCL (default=auto)]),[
-],[
-	with_opencl="no"
-])
-
-if test $with_opencl != "no" ; then {
-	AC_CHECK_HEADER(CL/opencl.h,,AC_MSG_ERROR([ ** You need to install opencl-headers package.]))
-	PKG_CHECK_MODULES(LIBCL, OpenCL,[
-		CONFIG_DEPS="$CONFIG_DEPS OpenCL"
-		with_opencl="yes"
-	],[
-		AC_MSG_ERROR([ ** You need to install OpenCL.])
-		with_opencl="no"
-	])
-	
-} ; fi
-AM_CONDITIONAL(WITH_OPENCL, test $with_opencl = yes)
-
 PKG_CHECK_MODULES(LIBFFTW, fftw3,,[
 	AC_MSG_ERROR([ ** You need to install FFTW 3.])
 ])
@@ -631,8 +585,6 @@ vImage ---------------------------> $with_vimage
 ImageMagick ----------------------> $with_imagemagick
 Magick++ -------------------------> $with_magickpp
 FFMPEG ---------------------------> $with_ffmpeg
-OpenGL ---------------------------> $with_opengl
-OpenCL ---------------------------> $with_opencl
 libdv ----------------------------> $with_libdv
 OpenEXR --------------------------> $with_openexr
 

